types/highland/index.d.ts

```diff
@@ -71,8 +72,9 @@ interface HighlandStatic {
 	 * @api public
 	 */
 	<R>(): Highland.Stream<R>;
+  <R>(xs: Highland.Stream<R>[]): Highland.Stream<R>;
 	<R>(xs: R[]): Highland.Stream<R>;
-	<R>(xs: (push: (err: Error, x?: R) => void, next: () => void) => void): Highland.Stream<R>;
+	<R>(xs: (push: (err: Error | null, x?: R) => void, next: () => void) => void): Highland.Stream<R>;
```

Comment:
It might be a good idea to add tests for these cases. What do you think?

Reply:
This declaration is corresponding to `Async` in http://highlandjs.org/#async

In fact, it should have covered by the test here
https://github.com/alvis/DefinitelyTyped/blob/471ce58849ca7a17de1b96f504ec08fc3f7219fa/types/highland/highland-tests.ts#L129

Reply:
Right! Now I'm wondering why it was working before. Why did [`push` accept `null`](https://github.com/alvis/DefinitelyTyped/blob/471ce58849ca7a17de1b96f504ec08fc3f7219fa/types/highland/highland-tests.ts#L130) as a first argument before your change?

Reply:
Interesting question. I wondered that too.

Reply:
I think it worked because `strictNullChecks` was only introduced recently.

Reply:
I think you are right.